src/lib/models/workflow-actions.ts

```diff
@@ -3,3 +3,9 @@ export enum ResetType {
   LastWorkflowTask = 1,
   EventId = 2,
 }
+
+export enum ResetReapplyType {
```

Comment:
Instead of adding an enum here, what do you think about using the proto definitions directly? E.g. 

```js
  import { temporal } from '@temporalio/proto';
  temporal.api.enums.v1.ResetReapplyType
```